test/ibmq/runtime/test_runtime.py

```diff
@@ -101,6 +104,24 @@ def test_coder(self):
         self.assertIsInstance(decoded_result, Result)
         self.assertTrue((decoded_array == orig_array).all())
 
+    def test_encoder_qc(self):
```

Comment:
nit - Since both encoder and decoder are tested here, could we rename this test accordingly? Suggestion: `test_encoder_decoder_qc`

Reply:
I changed it to `test_coder_qc` since we already have a `test_coder` test case.